### ISSUE TITLE
Fix README and other documentation

### ISSUE THREAD

**Issue by AlexWayfer** [pulsar-edit/pulsar#64]
### Prerequisites

* [x] Put an X between the brackets on this line if you have done all of the following:
    * Have you reproduced the issue in safe mode? : <https://flight-manual.atom.io/hacking-atom/sections/debugging/#using-safe-mode>
    * Have you used the debugging guide to fix any issues? : <https://flight-manual.atom.io/hacking-atom/sections/debugging/>
    * Checked our FAQs to make sure your question isn't answered there? : <https://discuss.atom.io/c/faq>
    * Does your issue already exist? : <https://github.com/issues?utf8=✓&q=is%3Aissue+user%3Apulsar>
    * Checked that there is not already an Pulsar package that provides the described functionality: <https://pulsar-edit.com/packages>

### Description

I still see "Atom" mentions in different places.

### Steps to Reproduce

Read the README, click some links.

**Expected behavior:**

Everything is about "Pulsar".

**Actual behavior:**

![image](https://user-images.githubusercontent.com/6510020/191859434-5ef1713d-666c-4d56-bb5e-c94e778a9b4c.png)

![image](https://user-images.githubusercontent.com/6510020/191859475-fa9500a5-1da9-4293-a16b-eff91ab9e6ff.png)

**How often does this happen:**

100%

### Versions

Current.

**Comment by kaosine** [comment]
These are eventually going to be replaced by our website content. We're working on getting that all up to date. I'm currently working on a pr for this over on pulsar-edit/pulsar-edit.github.io. It's just taking a bit because of some weird bugs and such from what we're doing to update everything.

**Comment by AlexWayfer** [comment]
> These are eventually going to be replaced by our website content. We're working on getting that all up to date. I'm currently working on a pr for this over on pulsar-edit/pulsar-edit.github.io. It's just taking a bit because of some weird bugs and such from what we're doing to update everything.
> 
> Especially all the mentions to atom and atom imaging we're waiting on final updates to logos that we're considering to replace that with.

Great. I've created the issue here because it's related to this repo. You can consider it as a part of replacement by site or whatever.

**Comment by softcode589** [comment]
Also, the Atom --> Pulsar on the header was done by me on my rebrand PR with this, I haven't removed the logo in that PR but maybe I'll replace (or remove) it in a new PR.

**Comment by kaosine** [comment]
I already commented out the logo for now. Left it until we replace the image so we can just change the urls.

**Comment by preland** [comment]
This is a slightly different direction than the original intent of this question, but I will say this here since creating a separate issue would create a duplicate "Fix README and docs" issue.

I understand that the sunsetting and multiple project and code transfers has made things difficult, but the project as a whole is in serious disarray. Even attempting to get a working binary or buildable file is a nightmare of bouncing between this github repo, the cirrus ci tracker, the atom-community repo, as well as random links to the original atom build/download instructions.

Once again, I understand why this is the case, but it is a serious problem. I would personally be willing to help, but I unfortunately don't know enough about the project to assist beyond very obvious "replace Atom with Pulsar" in random files, as this issue initially touched upon.

**Comment by Spiker985** [comment]
You can download pre-built binaries from the Cirrus badge in the pulsar-edit/pulsar/README.md

Build documentation is available at https://pulsar-edit.dev/docs/launch-manual/sections/core-hacking/

In the future, documentation will be on https://www.pulsar-edit.dev

**Comment by preland** [comment]
> You can download pre-built binaries from the Cirrus badge in the pulsar-edit/pulsar/README.md
> 
> Build documentation is available at https://pulsar-edit.dev/docs/launch-manual/sections/core-hacking/
> 
> In the future, documentation will be on https://www.pulsar-edit.dev

So I looked into this;

The installation badge links to docs/installation.md, which 1. Claims to have instructions to install Atom Community, and 2. has a link that redirects to an Azure pipeline for Atom Community. While the download technically works, and you can get a working binary of Atom (Atom Dev), I am fairly certain this has nothing to do with Pulsar.

The build documentation seems to be ok, although it is still missing portions of it, most notably dependency instructions for MacOS and ArchLinux (ironically, the only two OS's that I have on hand). 

As you said, documentation on pulsar-edit.dev is not yet completed, which is understandable. I would, however, suggest that the "download" button on the main page be removed or renamed, as it currently doesn't really do much more than confuse those who try to use it.

**Comment by confused-Techie** [comment]
So I think this is mostly resolved? Are there any locations where the rebrand isn't complete that need to be?

**Comment by AlexWayfer** [comment]
> So I think this is mostly resolved? Are there any locations where the rebrand isn't complete that need to be?

I believe so, thank you.

I also saw "Atom" on the welcome screen of the editor itself, but I think it's another issue.

Also I see "Atom" as a part of branch name on the README screenshot, but it's an unimportant thing. 😅

**Comment by confused-Techie** [comment]
> > So I think this is mostly resolved? Are there any locations where the rebrand isn't complete that need to be?
> 
> I believe so, thank you.
> 
> I also saw "Atom" on the welcome screen of the editor itself, but I think it's another issue.
> 
> Also I see "Atom" as a part of branch name on the README screenshot, but it's an unimportant thing. 😅

Yeah the screenshot should be updated, but the image on the welcome screen should be fixed now. Feel free to point out anywhere else you see it. But thanks for closing this one!
